ambari-server/src/main/java/org/apache/ambari/server/orm/entities/RequestEntity.java

```diff
@@ -125,6 +125,9 @@
   @Column(name = "exclusive_execution", insertable = true, updatable = true, nullable = false)
   private Integer exclusive = 0;
 
+  @Column(name = "user_name", nullable = false)
```

Comment:
We do set user of a BatchRequest as "internal" so this non-nullable might be ok but make sure to check other scenarios for this, IMO enforcing this as being non-nullable should not be needed.

Reply:
Thanks. I'll modify it tomorrow.